pkg/volume/local/local.go

```diff
@@ -204,8 +210,26 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
-		return nil
+	if fsGroup != nil {
+		refs, err := mount.GetMountRefsByDev(m.mounter, m.globalPath)
```

Comment:
You are absolutely right, do you want me to rename GetMountRefsByDev into something else to be more intuitive?